controller/controller.go

```diff
@@ -219,16 +219,57 @@ func (c *Controller) DeleteFunc(obj interface{}) {
 // ProcessEvents takes the event channels created by the operatorkit informer
 // and executes the controller's event functions accordingly.
 func (c *Controller) ProcessEvents(ctx context.Context, deleteChan chan watch.Event, updateChan chan watch.Event, errChan chan error) error {
+	loop := -1
+
 	operation := func() error {
 		for {
+			loop++
```

Comment:
Maybe dumb question but what is the loop variable for? Is it added to the context for debugging?

Reply:
OK that makes sense. Filtering the logs for the loop is nice 👍.